readthedocs_build/config/test_find.py

```diff
@@ -39,11 +39,11 @@ def test_find_nested(tmpdir):
     apply_fs(tmpdir, {'first/readthedocs.yml': ''})
 
     base = str(tmpdir)
-    paths = list(find_all(base, ('readthedocs.yml',)))
-    assert set(paths) == set([
+    paths = set(list(find_all(base, r'readthedocs\.yml')))
```

Comment:
We can omit the `list` conversion here and directly do a `set`

Reply:
yep